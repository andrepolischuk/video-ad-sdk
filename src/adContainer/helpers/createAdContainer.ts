const createAdContainer = (): HTMLDivElement => {
  const adContainer = document.createElement('div')

  adContainer.classList.add('mol-video-ad-container')
  adContainer.style.width = '100%'
  adContainer.style.height = '100%'

  return adContainer
}

export default createAdContainer
