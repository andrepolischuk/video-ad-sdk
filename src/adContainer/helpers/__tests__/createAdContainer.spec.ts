import createAdContainer from '../createAdContainer';

describe('createAdContainer', () => {
  test('must return a video ad container element', () => {
    const adContainer = createAdContainer();

    expect(adContainer).toMatchSnapshot();
  });
});
