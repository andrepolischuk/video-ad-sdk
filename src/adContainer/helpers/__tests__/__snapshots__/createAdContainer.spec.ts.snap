// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createAdContainer must return a video ad container element 1`] = `
<div
  class="mol-video-ad-container"
  style="width: 100%; height: 100%;"
/>
`;
